apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/[deploymentSlug]/releases/[versionId]/TargetReleaseTable.tsx

```
@@ -137,81 +148,221 @@ const CollapsibleTableRow: React.FC<CollapsibleTableRowProps> = ({
+                                    size="icon"
+                                    className="h-6 w-6"
+                                  >
+                                    <IconDots className="h-4 w-4" />
+                                  </Button>
+                                </JobDropdownMenu>
+                              </div>
+                            </TableCell>
+                          </TableRow>
+                        );
+                      })}
+                    </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)